ws-server/src/server.rs

```diff
@@ -107,7 +109,7 @@ async fn background_task(
 	socket: tokio::net::TcpStream,
 	methods: Arc<Methods>,
 	conn_id: ConnectionId,
-) -> Result<(), Error> {
+) -> Result<(), WsError> {
```

Comment:
shouldn't this be `Error` for a uniform API for the servers?

Reply:
For posterity: you are correct, the API should be consistent and return the same error type, i.e. `jsonrpsee_types::Error`. That would require a bunch of yucky `map_err` which would need to be annotated, e.g. `.map_err::<WsError, _>(Into::into)`.